cmd/cluster-init/main.go

```diff
@@ -220,9 +220,16 @@ func submitPR(o options) error {
 	return exec.Command("git", "checkout", master).Run()
 }
 
-func initClusterBuildFarmDir(o options) error {
+func updateClusterBuildFarmDir(o options) error {
 	buildDir := buildFarmDirFor(o.releaseRepo, o.clusterName)
-	logrus.Infof("Creating build dir: %s", buildDir)
+	if o.update {
+		logrus.Infof("Updating build dir: %s", buildDir)
+		if err := os.RemoveAll(buildDir); err != nil {
```

Comment:
If we remove all, it will remove all the contents that are manually added, right? Something like, 

```
init build03
add some.yaml into build03 folder manually
update build03: this will remove some.yaml
```

How are we going to recover it?

Reply:
yes, I was under the impression that we wouldn't want to be adding anything manually there. I guess we could just remove the 2 symlinks and leave the rest in tact. thoughts?

Reply:
Removing (and recreating later) the symlinks sounds good to me.

I think in general: if our tool expects
- a file (completed controlled by our tool) or a symlink, we remove and create
- a dir, we do nothing